conformity/fields/basic.py

```diff
@@ -202,6 +202,7 @@ class UnicodeString(Base):
     min_length = attr.ib(default=None)
     max_length = attr.ib(default=None)
     description = attr.ib(default=None)
+    not_empty = attr.ib(default=None)
```

Comment:
`default=False, type=bool`

Reply:
I was in doubt about that one, logically it makes sense to be bool, but then if it is, then `strip_none` will not filter it out as a flag that is not set

Reply:
I don't consider that a problem. The introspection won't be _wrong_. It will just always show whether or not blanks are allowed.

In fact, for that matter, now that I think about it, I would prefer this to be named `allow_empty` and default to `True`, or alternatively be named `disallow_empty` or `prohibit_empty` and default to `False`. `not_empty` strikes me as confusing.

Reply:
Initially I had it as `allow_blank`, but I wanted to avoid confusion with the Django's thing which _defaults_ to `False`. If you'd consider that fine, I'll change it back

Reply:
I do consider that fine.